internal/daemon/worker/status.go

```diff
@@ -230,7 +230,7 @@ func (w *Worker) sendWorkerStatus(cancelCtx context.Context, sessionManager sess
 		UpdateTags:                            w.updateTags.Load(),
 	})
 	if err != nil {
-		event.WriteError(cancelCtx, op, err, event.WithInfoMsg("error making status request to controller"))
```

Comment:
Since we call `.Load` already on line 185, is it possible to just assign that to a variable and use it in both places? Not sure what the load semantics are.

Reply:
I'll push up a change but an item to consider. Would this result in a thread safety/race condition issues?